Auto submodule update: cmake - scripts: Fix flakes

swiftnav-svc-jenkins opened the pull request:
Automated PR by Jenkins. If CI has passed successfully, merge away!

**cmake**  38138963  ->  2053cbd1
-  2053cbd1  :  scripts: Fix flakes


This pull request was created by https://jenkins.ci.swift-nav.com/job/CI%20Infra/job/submodule-update/7015/.

martin-swift commented:
Definitely.  I was wondering how that compiled; seems like nothing in the library itself is using it. 
Can you delete the useless comments while you at it?  (i.e. /* __cplusplus */)

mbavaro commented:
Indeed, no source file includes this..

martin-swift commented:
How did you find it?  is something in LSNP including it?

mbavaro commented:
yes, I am trying to integrate it

swiftnav-svc-jenkins committed:
cmake - scripts: Fix flakes
`sys` was unused.
Also tabs were used in memcheck_xml2junit_converter.py

Triggered-By:	cmake	2053cbd1ce36c1b33a6802550c40a23a41137d4e

jayvdb commented:
outdated
